README.md

```diff
@@ -1,4 +1,4 @@
-# data-viewer
+# front-end
```

Comment:
This was auto-populated by github and while we're here changing it it might be nice to rename the title to something more human like "Philanthropy Data Commons Front End" (the service is currently titled "Philanthropy Data Commons Service")

Reply:
I do think (whether in this PR or a followup) that re-titling the README is a good idea.

Reply:
Let's do that as part of the follow-up rebrand PR!

Reply:
FWIW I don't think the markdown title would be part of the rebrand since it is a dev document.  If there's a reason we don't want to change this to "Philanthropy Data Commons Front End" now that's fine, but I don't think this file should ever say "Philanthropy Data Commons Data Portal" or whatever we end up landing on for the branded name.

Reply:
I do agree this file's title will almost certainly differ from the landing page title, but I would reframe the follow-up PR from "branding" (even though it involves that) to "user-facing language", where users are both website visitors and devs reading documentation. In that case, both the language on the landing page and the README title are fair game (IMOoooocow).

Reply:
Whoops I edited out my soft-landing "What do you think?" when rearranging sentences, so…

What do you think?

Reply:
> I feel like the common convention is to have the title match the repo name

Got it!  I had always interpreted that as more of "it's the default and nobody got around to making a real title" BUT I'm also fine just punting / leaving as proposed!

Carry forth!  The bike shed is beautiful regardless!

Reply:
:bike: :house: 

I did a little further research and didn't find anything particularly persuasive in one direction or the other! So, now that I have your blessing, I'm going to pull the rip cord on this and leave it alone for now. :blue_heart: :parachute:

Reply:
> Carry forth! The bike shed is beautiful regardless!

Obligatory https://fuchsia.bikeshed.com/ .